doc/Makefile

```diff
@@ -5,7 +5,8 @@ default: html
 # You can set these variables from the command line.
 SRCDIR           := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME        := $(abspath $(SRCDIR)/..)
-JULIA_EXECUTABLE := $(JULIAHOME)/usr/bin/julia
+include $(JULIAHOME)/Make.inc
```

Comment:
do you run this often in a context where you couldn't set JULIA_EXECUTABLE to reflect the custom bindir?